db/db_test_util.cc

```diff
@@ -456,6 +465,19 @@ Options DBTestBase::CurrentOptions(
 #endif
       break;
     }
+    case kPipelinedWrite: {
+      options.enable_pipelined_write = true;
+      break;
+    }
+    case kConcurrentMemTableWrite: {
+      options.allow_concurrent_memtable_write = true;
+      break;
+    }
+    case kPipelinedWriteAndConcurrentMemTableWrite: {
+      options.enable_pipelined_write = true;
+      options.allow_concurrent_memtable_write = true;
```

Comment:
Since allow_concurrent_memtable_write  is true by default, I think this is the same as the previous case.